nss-day-cohort-52/rare-api-smoky-pythons
# Added property decorator to Post model and refactored retrieve and list methods in PostView class

### Issue ticket(s): #4 

### New file(s):

### Modified File(s): `post.py`

### Modifications:
- Added a property decorator to the Post model to show whether a post belongs to the current user or not

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to al-postTags
3. fetch posts in postman, you should see a new `is_owner` property on each post object that comes back (true/false)
